blockchain/objects/dkg_state.go

```diff
@@ -124,48 +89,120 @@ type DkgState struct {
 	// Inverse stores the multiplicative inverses
 	// of elements. This may be used in GPKJGroupAccusation logic.
 	Inverse []*big.Int // "
+}
+
+// GetSortedParticipants returns the participant list sorted by Index field
+func (state *DkgState) GetSortedParticipants() ParticipantList {
```

Comment:
need to make sure this has state.lock/unlock where used, see previous state locking comment.

Reply:
same for functions below

Reply:
https://madhive.atlassian.net/browse/MP-313